code/modules/shuttle/shuttle.dm

```diff
@@ -503,6 +503,8 @@
 			//move mobile to new location
 			for(var/atom/movable/AM in T0)
 				AM.onShuttleMove(T1, rotation)
+			
+			T1.onShuttleMove(T0)
```

Comment:
/atom/movable/proc/onShuttleMove and /turf/proc/onShuttleMove aren't the same function